cargo-apk/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+- Added `cargo apk check`. Useful for compile-testing crates that contain C/C++ dependencies or
+  target-specific conditional compilation, but do not provide a cdylib target.
+
```

Comment:
Oh, in hindsight I don't think we ever had newlines between bulletpoints in the changelog. Feel free to merge this and make a new release after dropping this one :)